### PR TITLE
Add fileName to the webpack loader params, allowing users to identify…

### PR DESCRIPTION
**Pull request by phil-lgr**
… the source file for vanilla-extra css imports

@mattcompiles

### CODE REVIEW

File: .changeset/nervous-beans-destroy.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@vanilla-extract/webpack-plugin': patch
+---
+
+Add fileName to the webpack loader params, allowing users to identify the source file for vanilla-extra css imports
```
Comment (mattcompiles) [inline .changeset/nervous-beans-destroy.md:5]:
```suggestion
Add fileName to the CSS virtual loader params

This allows users to identify the source file for vanilla-extract css imports
```

**Review by mattcompiles — APPROVED** [review]

**Comment by mattcompiles** [comment]
@phil-lgr Happy to merge this but the branch is out of date and you haven't given me write permissions to update your fork.

**Comment by phil-lgr** [comment]
Add you to the fork for future work

**Comment by phil-lgr** [comment]
@mattcompiles updated the branch, let me know if you could add me as collaborator, I'd be glad to send more PRs in the future.

**Review by mattcompiles — APPROVED** [review]

**Pull request merged**